backend/app/services/chat/storage/task_manager.py

```
@@ -636,6 +636,84 @@ async def create_task_and_subtasks(
+        # This is a simplified approach - we get basic model info from team
+        model_config = None
+        try:
+            from app.services.chat.config import ChatConfigBuilder
+
+            config_builder = ChatConfigBuilder(db, team, user.id, user.user_name)
+            chat_config = config_builder.build(
+                override_model_name=params.model_id,
+                force_override=params.force_override_bot_model,
+            )
+            model_config = chat_config.model_config
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)